### PR TITLE
Expand qualification abbreviations on first instance eg pgce and qts

### PR DESCRIPTION
**Pull request by avinhurry**
### Context

We are changing our course qualification abbreviations for the first instance. 

`Note: After merging this PR: https://github.com/DFE-Digital/publish-teacher-training/pull/3148 a rebase should allow the smoke tests on this PR to pass, as the new method is defined there.`

### Changes proposed in this pull request

- Utilise new method on course show page, defined in this PR: https://github.com/DFE-Digital/publish-teacher-training/pull/3148

- Describe qualification abbreviations for assistive technologies and maintain usability for sighted users on results page

### Guidance to review

- Pull down this PR and the [respective PR](https://github.com/DFE-Digital/publish-teacher-training/pull/3148) for Publish locally. 
- Navigate to courses with different qualifications
- Ensure the expanded description is correct.

### Screenshots

image
image
image
image
image

_________________________________________________________________________________________________________

image

image
image
image
image

### Trello card

https://trello.com/c/yVRt6p0N/828-expand-abbreviations-on-first-instance-eg-pgce-and-qts

### Checklist

- [x] Rebased `main`
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by avinhurry** [comment]
[Expand abbreviations on first instance (e.g., PGCE and QTS)](https://trello.com/c/yVRt6p0N/828-expand-abbreviations-on-first-instance-eg-pgce-and-qts)

File: app/components/results/search_result_component.html.erb [hunk 1]
```diff
@@ -14,7 +14,7 @@
         <p class="govuk-body"><%= course.qualification.humanize %></p>
         <p class="govuk-hint"><%= accredited_body %></p>
       <% else %>
-        <%= course.qualification.humanize %>
+       <%= qualification_description_with_abbreviation[course.qualification].html_safe %>
```
Comment (defong) [inline app/components/results/search_result_component.html.erb:17]:
```
t('find.qualification.description_with_abbreviation.#{course.qualification}')
```

Comment (avinhurry) [reply app/components/results/search_result_component.html.erb]: https://github.com/DFE-Digital/find-teacher-training/pull/1620/commits/4902efc7c4a7f00d2826c47db8902c1171a08b23

File: app/components/results/search_result_component.html.erb [hunk 2]
```diff
@@ -14,7 +14,7 @@
         <p class="govuk-body"><%= course.qualification.humanize %></p>
         <p class="govuk-hint"><%= accredited_body %></p>
       <% else %>
-        <%= course.qualification.humanize %>
+       <%=t("find.qualification.description_with_abbreviation.#{course.qualification}").html_safe %>
```
Comment (defong) [inline app/components/results/search_result_component.html.erb:17]:
```suggestion
       <%=t("find.qualification.description_with_abbreviation.#{course.qualification}.html") %>
```

https://guides.rubyonrails.org/i18n.html#using-safe-html-translations

**Review by defong — APPROVED** [review]

File: app/components/results/search_result_component.rb [hunk 3]
```diff
@@ -52,5 +52,15 @@ def accredited_body
 
       "QTS ratified by #{helpers.smart_quotes(course['accrediting_provider']['provider_name'])}"
     end
+
+    def qualification_description_with_abbreviation
```
Comment (defong) [inline app/components/results/search_result_component.rb:56]: Ummm?

**Review by defong — CHANGES_REQUESTED** [review]

File: app/components/results/search_result_component.rb (hunk 3 above)
Comment (avinhurry) [reply app/components/results/search_result_component.rb:56]: lol

File: app/components/results/search_result_component.html.erb (hunk 2 above)
Comment (avinhurry) [reply app/components/results/search_result_component.html.erb:17]: 🤯

Comment (avinhurry) [reply app/components/results/search_result_component.html.erb]: https://github.com/DFE-Digital/find-teacher-training/pull/1620/commits/f099056756c4f9dae7882ec5f881eec805799b49

Whats the benefit of this anyways, over `.html_safe`

File: app/components/results/search_result_component.rb (hunk 3 above)
Comment (avinhurry) [reply app/components/results/search_result_component.rb]: 🪄

**Review by defong — APPROVED** [review]

**Comment by simonwhatley** [comment]
The page heading and lede paragraph containing the qualification should be in a two-thirds column like this:

```html
<div class="govuk-grid-column-two-thirds">
  <h1 class="govuk-heading-xl">
    <span class="govuk-caption-xl">
      Academies Enterprise Trust: Essex
    </span>
    Biology (K599)
  </h1>
  <p class="govuk-body-l">Postgraduate certificate in education (PGCE) with qualified teacher status (QTS)</p>
</div>
```

Note the provider name should also be a _proper_ caption `govuk-caption-xl`.

**Review by simonwhatley — APPROVED** [review]

**Pull request merged**